### PR TITLE
update: thelounge configuration example

### PR DESCRIPTION
**Pull request by ishanjain28**

### CODE REVIEW

File: example_configs/thelounge.md
Comment (nitnelave) [inline example_configs/thelounge.md:14]: What's the root DN? And why ldap-editor? Maybe we need to describe this a bit more

Comment (ishanjain28) [reply example_configs/thelounge.md:14]: rootDN/rootPassword are the credentials used to login to lldap and perform queries. (Similar to bind user and bind user password. the lounge uses a different name for it) 

`ldap-editor` user is a member of `lldap_strict_readonly` and `lldap_password_manager` groups.

Comment (nitnelave) [reply example_configs/thelounge.md:14]: Right, this would be great to put in the documentation :)

**Comment by ishanjain28** [comment]
hey @nitnelave Please let me know if you have any more feedback on this PR. It is complete from my end and awaiting your approval

**Comment by nitnelave** [comment]
@ishanjain28  Sorry, my comment was stuck in "pending", I never actually sent it.

File: example_configs/thelounge.md
Comment (ishanjain28) [reply example_configs/thelounge.md:23]: That makes sense and I'll make this change.

**Comment by nitnelave** [comment]
Hey @ishanjain28 , do you want to update this PR, or should I take it over?

File: example_configs/thelounge.md
Comment (nitnelave) [inline example_configs/thelounge.md:5]: Make it a title, with 2 `#` before

File: example_configs/thelounge.md
Comment (nitnelave) [inline example_configs/thelounge.md:17]: I would put the description first, before the config.

Also, all users are members of the base DN, so we don't need to specify that.

File: example_configs/thelounge.md
Comment (nitnelave) [inline example_configs/thelounge.md:20]: Title

File: example_configs/thelounge.md
Comment (nitnelave) [inline example_configs/thelounge.md:38]: Can you reset your password via the lounge? Also, the readonly permission is redundant, password manager is a superset.

**Review by nitnelave — CHANGES_REQUESTED** [review]

**Comment by ishanjain28** [comment]
Hey @nitnelave Apologies for the delays. 

I have updated the config into configs for simple and more advanced setups and rebased the PR. 

> The root DN is the bind user, preferably a member of strict readonly.

This was a great suggestion! I assumed the lounge supported changing passwords for users but they do not so we should not use a user with the permission to change passwords in this context, Just the user with read permissions is sufficient.

File: example_configs/thelounge.md
Comment (ishanjain28) [reply example_configs/thelounge.md:38]: @nitnelave Yep, Updated this in the next commit

**Comment by ishanjain28** [comment]
I believe all of the comments have been addressed. I am not sure why github's UI is still stuck with "1 change requested"

**Review by nitnelave — APPROVED** [review]
Great, thanks!

**Comment by nitnelave** [comment]
It was just waiting for my approval :)

**Comment by nitnelave** [comment]
Thank you everyone for the contribution!
This feature is now available in the latest release, [v0.5.0](https://github.com/lldap/lldap/releases/tag/v0.5.0).
You can support LLDAP by starring our repo, contributing some configuration examples and becoming a sponsor.

**Pull request merged**
